README.md

```diff
@@ -1,20 +1,33 @@
-HumHub - Social Network Kit
+HumHub - Putting People and Pieces together
```

Comment:
I'm not sure about this, maybe something along the lines of;
` HumHub - Bringing People and Communities Together` or `HumHub - Creating Communities Together `?

Reply:
We use `Putting People and Pieces together` as claim